test/auth.js

```diff
@@ -18,16 +19,51 @@ describe('auth interface', () => {
         .then((resp) => should(resp.uri).startWith('safe-auth:'));
   });
 
+  it('throws error if missing permissions object', async () => {
```

Comment:
I don't think this test is correct, we should allow to call this function without any permissions object.

Reply:
ahh right. thanks for catching this. Should still be returning auth URI.